src/style.css

```
@@ -0,0 +1,93 @@
+* {
+  margin: 0;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+}
+
+.body {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  height: 300px;
+  flex-basis: 33.33%;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  min-width: 300px;
+ 
+}
+
+.block--1 {
+  order: 1;
+  background-color: rgb(255 0 0);
+}
```

you don't need custom orders here, remove them from here, just change them in your media rules
```suggestion
.block--1 {
  background-color: rgb(255 0 0);
}
```